src/reference/faq.rst

```diff
@@ -0,0 +1,29 @@
+================================
+Frequently Asked Questions (FAQ)
+================================
+
+.. TODO: Add a table of contents once there are more sections. Doesn't seem necessary at the time of writing.
+
+Installation
+============
+
+There are many ways to install Nextstrain, and we aim to simplify the installation guide so it is easy to follow along. Here, you will find answers to some common questions about the installation process.
+
+.. _why-intel-miniconda-installer-on-apple-silicon:
+
+Why recommend the Intel Miniconda installer for Mac computers with Apple silicon (e.g. M1)?
```

Comment:
Reworded to reference "Apple silicon" instead of M1 for future-proofing.

Reply:
Yes, good point!